core/src/main/java/io/openepcis/epc/eventhash/ConstantEventHashInfo.java

```diff
@@ -68,40 +71,94 @@ public class ConstantEventHashInfo {
+      List.of("epcis:EPCISDocument", "EPCISBody", "EventList");
   protected static final List<String> WHAT_DIMENSION_XML_PATH =
-      Arrays.asList("epc/epcList/", "epc/childEPCs/", "epc/inputEPCList/", "epc/outputEPCList");
+      List.of("epc/epcList/", "epc/childEPCs/", "epc/inputEPCList/", "epc/outputEPCList");
   protected static final List<String> WHY_DIMENSION_XML_PATH =
-      Arrays.asList(
+      List.of(
           "bizTransaction/bizTransactionList", "source/sourceList", "destination/destinationList");
   protected static final List<String> HOW_DIMENSION_XML_PATH =
-      Arrays.asList(
+      List.of(
           "sensorMetadata/sensorElement/sensorElementList",
           "sensorReport/sensorElement/sensorElementList");
   protected static final List<String> USER_EXTENSION_WRAPPER =
-      Arrays.asList("sensorElementList", SENSOR_ELEMENT);
+      List.of(SENSOR_ELEMENT_LIST, SENSOR_ELEMENT);
+
+  protected static final List<String> EXCLUDE_LINE_BREAK =
+      List.of(
+          "epcList",
+          "childEPCs",
+          "inputEPCList",
+          "outputEPCList",
+          "quantityList",
+          "childQuantityList",
+          "inputQuantityList",
+          "outputQuantityList",
+          QUANTITY_ELEMENT,
+          "readPoint",
+          "bizLocation",
+          "bizTransactionList",
+          "sourceList",
+          "destinationList",
+          SENSOR_ELEMENT_LIST,
+          SENSOR_ELEMENT,
+          "sensorMetadata",
+          SENSOR_REPORT,
+          "type",
+          PERSISTENT_DISPOSITION,
+          "bizTransaction",
+          "ilmd");
+
+  protected static final List<String> SHORTNAME_FIELDS =
+      List.of(
+          "epc",
+          "epcClass",
+          "id",
+          "deviceID",
+          "deviceMetadata",
+          "rawData",
+          "dataProcessingMethod",
+          "bizRules",
+          "microorganism",
+          "chemicalSubstance",
+          "coordinateReferenceSystem",
+          "uriValue");
+
+  protected static final Map<String, String> SENSOR_REPORT_FORMAT = new HashMap<>();
 
   static {
     BARE_STRING_FIELD_PARENT_CHILD.put("bizStep", "bizStep");
     BARE_STRING_FIELD_PARENT_CHILD.put("disposition", "disposition");
     BARE_STRING_FIELD_PARENT_CHILD.put("bizTransactionList", "type");
-    BARE_STRING_FIELD_PARENT_CHILD.put("persistentDisposition", "set");
-    BARE_STRING_FIELD_PARENT_CHILD.put("persistentDisposition", "unset");
+    BARE_STRING_FIELD_PARENT_CHILD.put(PERSISTENT_DISPOSITION, "set");
+    BARE_STRING_FIELD_PARENT_CHILD.put(PERSISTENT_DISPOSITION, "unset");
     BARE_STRING_FIELD_PARENT_CHILD.put("sourceList", "type");
     BARE_STRING_FIELD_PARENT_CHILD.put("destinationList", "type");
-    BARE_STRING_FIELD_PARENT_CHILD.put("errorDeclaration", "reason");
+
+    SENSOR_REPORT_FORMAT.put("type", GS1_DOMAIN);
+    SENSOR_REPORT_FORMAT.put("exception", GS1_DOMAIN + "SensorAlertType-");
+    SENSOR_REPORT_FORMAT.put("component", GS1_COMPONENT_DOMAIN);
```

Comment:
rename GS1_COOMPENENT_DOMAIN to GS1_CBV_DOMAIN and append "Comp" here.
This will make the static field more generic and I fear that we still might have missed something from cbv context.